# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../odyssey/gcc43.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- odyssey-0.5/src/ConfigFile.cxx.vanilla	2009-04-12 17:06:11.000000000 +0200
+++ odyssey-0.5/src/ConfigFile.cxx	2009-04-12 17:07:38.000000000 +0200
@@ -19,6 +19,7 @@
 using namespace std;
 
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
 #include <stdexcept>
 #include <new>
--- odyssey-0.5/src/device/PicDevice.cxx.vanilla	2009-04-12 17:08:05.000000000 +0200
+++ odyssey-0.5/src/device/PicDevice.cxx	2009-04-12 17:08:15.000000000 +0200
@@ -19,6 +19,7 @@
 using namespace std;
 
 #include <stdio.h>
+#include <string.h>
 #include <stdexcept>
 #include "config.h"
 #include "PicDevice.h"
--- odyssey-0.5/src/main.cxx.vanilla	2009-04-12 17:07:13.000000000 +0200
+++ odyssey-0.5/src/main.cxx	2009-04-12 17:07:25.000000000 +0200
@@ -19,6 +19,7 @@
 using namespace std;
 
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
 #include <signal.h>
 #include <unistd.h>
